Added time and space complexity to N-Queens problem

janos-laszlo opened the pull request:
Hello, I have added the time and space complexity to the N-Queens problem as asked in issue #8.

P.S.:This is my first contribution, sorry if I haven't done something right.

janos-laszlo committed:
Add time and space complexity to N-Queens problem

janos-laszlo committed:
Update N-Queens.cpp

janos-laszlo committed:
Time complexity layout fix

janos-laszlo committed:
Merge branch 'master' of https://github.com/janos-laszlo/Algos

janos-laszlo committed:
Calculated time and space complexity of N-Queens problem

faheel reviewed (changes requested):
Underline the "headings" by dashes (`-`) and remove the trailing `:`

janos-laszlo committed:
Underlined headings and removed trailing colons

In the N-Queens.cpp file, in the description of the problem,  I have underlined headings and removed the trailing colons after the headings.

janos-laszlo committed:
Tab error with indentation in the heading

janos-laszlo commented:
Hope this is what you asked :)

faheel commented:
Thanks for helping out! 😀
